apps/explorer/views.py

```diff
@@ -68,3 +72,39 @@ def get_queryset(self):
             'pixels__omics_unit__type',
             'pixels__omics_unit__strain__species',
         )
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context.update({
+            'export_form': PixelSetExportForm(),
+        })
+        return context
+
+
+class PixelSetExportView(LoginRequiredMixin, FormView):
+    ATTACHEMENT_FILENAME = 'pixelsets.zip'
+
+    form_class = PixelSetExportForm
+
+    def form_valid(self, form):
+        zip = export_pixelsets(form.cleaned_data['pixel_sets'])
```

Comment:
I would name this variable `archive` to avoid any confusion even if the zip module is called `zipfile`.